general/g.version/g.version.md

```diff
@@ -47,24 +47,65 @@ libgis revision: c9e8576cf
 libgis date: 2024-04-27T09:38:49+00:00
 ```
 
-### Full info in shell script style
-
-```sh
-g.version -rge
-
-version=8.4.0
-date=2024
-revision=d57f40906
-build_date=2024-05-23
-build_platform=x86_64-pc-linux-gnu
-build_off_t_size=8
-libgis_revision=c9e8576cf
-libgis_date=2024-04-27T09:38:49+00:00
-proj=8.2.1
-gdal=3.4.3
-geos=3.9.2
-sqlite=3.36.0
-```
+### Full info
+
+<!-- markdownlint-disable MD046 -->
+=== "Command line"
+
+    ```sh
+    g.version -re format=shell
+    ```
+
+    Possible output:
+
+    ```text
+    version=8.4.0
+    date=2024
+    revision=d57f40906
+    build_date=2024-05-23
+    build_platform=x86_64-pc-linux-gnu
+    build_off_t_size=8
+    libgis_revision=c9e8576cf
+    libgis_date=2024-04-27T09:38:49+00:00
+    proj=8.2.1
+    gdal=3.4.3
+    geos=3.9.2
+    sqlite=3.36.0
+    ```
+
+=== "Python (grass.script)"
+
+    ```python
+    import grass.script as gs
+
+    # Run g.version with JSON output and the -re flags
+    data = gs.parse_command(
+        "g.version",
+        flags="re",
+        format="json",
+    )
+
+    for key, value in data.items():
+        print(f"{key}: {value}")
+    ```
+
+    Possible output:
+
+    ```text
+    version: 8.4.0
+    date: 2024
+    revision: d57f40906
+    build_date: 2024-05-23
+    build_platform: x86_64-pc-linux-gnu
+    build_off_t_size: 8
+    libgis_revision: c9e8576cf
+    libgis_date: 2024-04-27T09:38:49+00:00
+    proj: 8.2.1
+    gdal: 3.4.3
+    geos: 3.9.2
+    sqlite: 3.36.0
```

Comment:
Here is a suggestion to simplify the example and include the json output


```suggestion
    print(data["version"])
    ```

    Possible output:

    ```text
    8.4.0
    ```

    Output JSON may look like this: 

    ```json
    {
        "version": "8.4.0",
        "date": "2024",
        "revision": "d57f40906",
        "build_date": "2024-05-23",
        "build_platform": "x86_64-pc-linux-gnu",
        "build_off_t_size": 8,
        "libgis_revision": "c9e8576cf",
        "libgis_date": "2024-04-27T09:38:49+00:00",
        "proj": "8.2.1",
        "gdal": "3.4.3",
        "geos": "3.9.2",
        "sqlite": "3.36.0"
    }
    ```